src/Middleware/PageDecorationMiddleware.php

```diff
@@ -35,26 +35,35 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
         $loris   = $request->getAttribute("loris");
         $page    = $request->getAttribute("pageclass")
         ?? new \NDB_Page($loris, new \NullModule($loris), "", "", "", "");
+
+        $DB            = \NDB_Factory::singleton()->database();
+        $language_rows = $DB->pselect("SELECT language_code, language_label FROM language", []);
+        foreach ($language_rows as $language_row) {
+            $supported_languages[] = [
+                                      'code' => $language_row['language_code'],
+                                      'name' => $language_row['language_label'],
+                                     ];
```

Comment:
![image](https://github.com/user-attachments/assets/b4199b66-7f4c-4ee4-b24e-600adf4d2ea7)

Reply:
Not advocating for a rule change here (because it will unfortunately not happen), but still wanted to speak my mind.